### ISSUE TITLE
Error when using stdout assume profile

### ISSUE THREAD

**Issue by meyerjrr** [common-fate/granted#316]
From @ttunua in Slack:

I changed some config for browser now it seems to keep trying to use firefox or a browser and ends up failing on x509 certs.
I changed the config back to stdout, tried calling unset.

Updating to latest version did not fix
![image (1)](https://user-images.githubusercontent.com/33018450/204711573-40439ac4-b44c-4c40-af37-a677781e106a.png)

Granted config file looks like this:
```
DefaultBrowser = "STDOUT" 
CustomBrowserPath = ""                                                                                                                                                                                                                                CustomSSOBrowserPath = ""
LastCheckForUpdates = 0
Ordering = ""
ExportCredentialSuffix = ""   
```

Running wsl ( ubuntu22.04 )

**Comment by ttunua** [comment]
We can close this.

I was messing around trying to see if I could get granted to launch Windows Firefox from WSL2 ( by configuring granted to use firefox with a path to firefox.exe ). This kind of worked in that assume would launch firefox to the install page for the extension but assume didn't work.

I was initially thrown this behaviour where `Command '/usr/bin/x-www-browser' requires the firefox snap to be installed.`
would appear. 

![image](https://user-images.githubusercontent.com/17945539/204733886-22cc50a6-5aaa-462b-a1a9-0d8ec5f32e51.png)

Since I had been messing with the default browser setting and installing some things along the way it appeared to me that somehow granted/assume was still trying to use some kind of default browser as I was used to seeing granted fail through the options (below)
![image](https://user-images.githubusercontent.com/17945539/204735148-7aee4fa5-8fee-4d50-a269-23d40a44ec8f.png)


This leads to what the actual issue is... Zscaler certs

- When I attempted to download the latest version of Granted, curl failed due to a cert error and I had to use -k to download
- I then remembered that we had a script to update-ca-certificates with Zscaler 
- Upon running the script Curl works normally and granted/assume work correctly

( This is the first time I have had to update certs for zscaler with everything having worked fine previously )



I had an older screenshot from when I was testing granted earlier.
If we compare the below to the first image it becomes clear.

above - Action=getSigninToken was being blocked by Zscaler
below - Action=login ( from a month or two ago )

![image](https://user-images.githubusercontent.com/17945539/204735482-edc3814b-981e-4364-945e-548fca4b15de.png)
